src/app/client/src/app/plugins/location/components/location-selection/location-selection.component.scss

```diff
@@ -1,337 +1,343 @@
-@import "variables";
-@import "mixins/mixins";
-@import 'pages/mat-modal';
-
-.sb-onboard {
-  sb-form::ng-deep {
-    .sb-textbox {
-      font-size: 14px;
-    }
-    .caret-down{
-      z-index: 1 !important;
-    }
-
-    .list-border {
-      padding: 16px !important;
-      overflow-y: auto;
-      position: relative;
-    }
-
-    ul {
-      padding: 0;
-      top: calculateRem(8px);
-
-      &[disabled] {
-        opacity: initial !important;
-      }
-    }
-
-    li {
-      margin: 0 !important;
-      padding: 16px !important;
-    }
-
-    li.placeholder {
-      margin: 0 !important;
-      padding: 0 !important;
-    }
-
-    .sb-modal-dropdown-web {
-      ul{
-        overflow-y: auto !important;
-        overflow: hidden;
-      }
-      li {
-        margin: 0 !important;
-        padding: 0 !important;
-
-        > div {
-          cursor: pointer;
-          padding: 16px;
-        }
-      }
-    }
-
-    .multi-select-container {
-      margin-top: 8px;
-    }
-  }
-
-  &__container {
-    width: 100%;
-    height: 100vh;
-    background: var(--primary-100);
-    padding: calculateRem(24px);
-    margin: 0 auto;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: space-between;
-    position: relative;
-    overflow-y: scroll;
-
-    @include respond-above(sm) {
-      border-radius: calculateRem(8px);
-      max-width: calculateRem(1008px);
-      min-height: calculateRem(610px);
-      height: auto;
-    }
-  }
-
-  &__header {
-    width: 100%;
-    text-align: center;
-    max-width: calculateRem(328px);
-
-    &::ng-deep h1 {
-      color: var(--primary-400);
-      font-size: calculateRem(24px);
-      font-weight: bold;
-      letter-spacing: 0;
-      line-height: calculateRem(32px);
-      text-align: center;
-      margin: calculateRem(8px) 0px;
-    }
-  }
-
-  &__content {
-    width: calculateRem(648px);
-    max-width: 100%;
-    margin-bottom: auto;
-  }
-
-  &__footer {
-    margin-top: calculateRem(16px);
-    width: 100%;
-    text-align: center;
-
-    @include respond-above(sm) {
-      max-width: 80%;
-    }
-  }
-  
-  ::ng-deep{
-    .brand-logo {
-      width: calculateRem(86px);
-    }
-  }
-
-  .arrow-icon {
-    float: right;
-
-    html[dir="rtl"] & {
-      float: left;
-    }
-  }
-
-  .close-modal {
-    position: absolute;
-    right: calculateRem(16px);
-    top: calculateRem(16px);
-    z-index: 9;
-    min-width: auto;
-  }
-
-  .back-btn {
-    position: absolute;
-    left: calculateRem(8px);
-    top: calculateRem(16px);
-    min-width: auto;
-  }
-}
-
 ::ng-deep {
-  p.subtitle {
-    color: var(--gray-800);
-    font-size: calculateRem(18px);
-    font-weight: bold;
-    letter-spacing: 0;
-    line-height: calculateRem(26px);
-    text-align: center;
+  html[layout=joy] app-location-selection .sb-mat__dropdown.mat-form-field-appearance-fill .mat-form-field-flex {
+    border-color: var(--gray-200);
   }
 }
 
```

Comment:
In place of commenting please delete the same if you need then it will be there in the git